modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -757,6 +757,64 @@ test.describe('Manage object fields through Model Builder', () => {
 });
 
 test.describe('Manage objectFields through Objects Admin UI', () => {
+	test("can check that it's possible to update created custom fields in a System Object", async ({
+		apiHelpers,
+		objectFieldsPage,
+		page,
+	}) => {
+		const objectDefinitionField =
+			await apiHelpers.buildRestClient(ObjectFieldAPI);
+
+		const fieldName = 'Custom Field';
+
+		const {items} = await apiHelpers.objectAdmin.getAllObjectDefinitions();
+
+		const systemObjectDefinition = items.find((item: ObjectDefinition) => {
+			return item.system === true;
+		});
+
+		await objectDefinitionField.postObjectDefinitionObjectField(
+			systemObjectDefinition.id,
+			{
+				DBType: 'String',
+				businessType: 'Text',
+				indexed: true,
+				label: {en_US: fieldName},
+				localized: false,
+				name: 'customField',
+				readOnly: 'false',
+				required: false,
+				state: false,
+			}
+		);
+
+		await objectFieldsPage.goto(systemObjectDefinition.label.en_US);
+
+		await objectFieldsPage.openObjectField(fieldName);
+
+		const labelUpdatedName = 'Custom Field Update';
+
+		await page
+			.frameLocator('iframe')
+			.getByPlaceholder('Text to translate...')
+			.fill(labelUpdatedName);
```

Comment:
To make the test more semantic, you could try using the `getByLabel('label')` selector. This will make it clearer to readers that we are filling in the label field.

Reply:
Hi @DavysonMelo, you will be able to do this if you pass the ID property to `InputLocalized`:

![image](https://github.com/user-attachments/assets/f722bf40-dff3-49b3-9146-91f88fffba92)